lib/python/qmk/cli/new/keyboard.py

```diff
@@ -143,36 +143,41 @@ def augment_community_info(src, dest):
 
 def prompt_keyboard():
     prompt = """{fg_yellow}Name Your Keyboard Project{style_reset_all}
-
 For more infomation, see:
 https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=naming-your-keyboardproject
 
-keyboard Name? """
+Keyboard Name? """
 
-    return question(prompt, validate=lambda x: not keyboard(x).exists())
+    kb_name = False
+    while not kb_name:
+        kb_name = question(prompt)
+        if keyboard(kb_name).exists():
+            prompt = f'Keyboard {{fg_cyan}}{kb_name}{{fg_reset}} already exists! Please choose a different name:'
+            kb_name = False
+    return kb_name
 
 
 def prompt_user():
-    prompt = """{fg_yellow}Attribution{style_reset_all}
-
+    prompt = """{fg_cyan}=========={fg_reset}
```

Comment:
Maybe its just the colour scheme i use, but im not getting much visual difference with this addition. The title text being a different colour to me is already enough to know its a new section.

Reply:
Fair enough, removed the "dividers".